testproject/Assets/Tests/Runtime/MultiprocessRuntime/BaseMultiprocessTests.cs

```diff
@@ -69,13 +78,27 @@ public virtual void SetupTestSuite()
 
         private void OnSceneLoaded(Scene scene, LoadSceneMode mode)
         {
-            MultiprocessLogger.Log($"OnSceneLoaded {scene.name}");
             SceneManager.sceneLoaded -= OnSceneLoaded;
+            var ushortport = ushort.Parse(m_Port);
+
+            var transport = NetworkManager.Singleton.NetworkConfig.NetworkTransport;
+            switch (transport)
+            {
+                case UNetTransport unetTransport:
```

Comment:
unet is deprecated, any reason we're not testing UTP?

Reply:
I didn't make this switch as that wasn't part of the scope of the work. However, I can flip it over if you like.

Reply:
This is now gone and UTP is default.

Reply:
Switching to UTP was non-trivial as the tests all failed when I tried to switch. I'm probably just doing it wrong. I'll address this later.

Reply:
ok, resolving this.
quick note: you can look at boss room to see what settings we used there. UTP's defaults aren't great :(